src/midi-player.ts

```diff
@@ -58,6 +60,9 @@ export class MidiPlayer implements IMidiPlayer {
             throw new Error('The player is already stopped.');
         }
 
+        // Bug #1: Chrome does not yet implement the clear() method.
```

Comment:
Might be good to include a link to the actual bug ticket https://issues.chromium.org/issues/40411677 

But looking at the compatibility table https://developer.mozilla.org/en-US/docs/Web/API/MIDIOutput/clear#browser_compatibility , it may be pointless to write expectations for Chrome / Canary specifically. We're still a long way from universal support.

Reply:
Thanks a lot @infojunkie for taking the time to review this. I really appreciate it.

I added a link to the issue on the Chromium tracker.

The main reason why I split the tests for Chrome and Chrome Canary is because I already did this elsewhere and it's easier for me to maintain the test setup when it's similar on all my repos. I have for example a script which occasionally updates the Karma configuration across all my repos.